IFC 4.3/SpanAnnotation_1/UT_SpanAnnotation_1.ifc

```diff
@@ -0,0 +1,75 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition []'),'2;1');
+FILE_NAME(
+/* name */ 'AlignmentWithSpanAnnotation.ifc',
+/* time_stamp */ '2020-09-15T08:25:20',
+/* author */ ('Sebastian Esser'),
+/* organization */ ('Chair of Computational Modeling and Simulation, Technical University of Munich'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.1.0 by Geometry Gym Pty Ltd built 2020-09-11T22:57:18',
+/* originating_system */ 'ConsoleCreateSpanAnnotation v1.0.0.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3RC1'));
```

Comment:
```suggestion
FILE_SCHEMA (('IFC4X3_RC1'));
```

Reply:
@jmirtsch would you mind to update your lib according to this change request with one of the next releases?